solutions/standard/outputs.tf

```diff
@@ -14,12 +14,12 @@ output "secrets_manager_guid" {
 }
 
 output "secrets_manager_id" {
-  description = "ID of Secrets Manager instance"
+  description = "ID of Secrets Manager instance. Same value as secrets_manager_guid"
   value       = var.existing_secrets_manager_crn == null ? module.secrets_manager[0].secrets_manager_id : local.secrets_manager_guid
```

Comment:
data lookup shows the following
```
 crn                     = "crn:v1:bluemix:public:secrets-manager:us-south:a/190c293e9fda4c6684b5acf4b17871b8:6bf6666d-73a6-41ad-a9ae-86cf9c9953e0::"
      + extensions              = {
          + "virtual_private_endpoints.dns_domain"             = "private.us-south.secrets-manager.appdomain.cloud"
          + "virtual_private_endpoints.dns_hosts.#"            = "1"
          + "virtual_private_endpoints.dns_hosts.0"            = "6bf6666d-73a6-41ad-a9ae-86cf9c9953e0"
          + "virtual_private_endpoints.endpoints.#"            = "3"
          + "virtual_private_endpoints.endpoints.0.ip_address" = "166.9.228.16"
          + "virtual_private_endpoints.endpoints.0.zone"       = "dal10"
          + "virtual_private_endpoints.endpoints.1.ip_address" = "166.9.229.17"
          + "virtual_private_endpoints.endpoints.1.zone"       = "dal12"
          + "virtual_private_endpoints.endpoints.2.ip_address" = "166.9.230.13"
          + "virtual_private_endpoints.endpoints.2.zone"       = "dal13"
          + "virtual_private_endpoints.origin_type"            = "cse"
          + "virtual_private_endpoints.ports.#"                = "1"
          + "virtual_private_endpoints.ports.0.port_max"       = "443"
          + "virtual_private_endpoints.ports.0.port_min"       = "443"
        }
      + guid                    = "6bf6666d-73a6-41ad-a9ae-86cf9c9953e0"
      + id                      = "crn:v1:bluemix:public:secrets-manager:us-south:a/190c293e9fda4c6684b5acf4b17871b8:6bf6666d-73a6-41ad-a9ae-86cf9c9953e0::"
      + identifier              = "crn:v1:bluemix:public:secrets-manager:us-south:a/190c293e9fda4c6684b5acf4b17871b8:6bf6666d-73a6-41ad-a9ae-86cf9c9953e0::"
      + location                = "us-south"
```
id is not similar to guid. Is that just in that case?

maybe we could use ?
`value       = var.existing_secrets_manager_crn == null ? module.secrets_manager[0].secrets_manager_id : data.ibm_resource_instance.existing_sm[0].id`

Reply:
Correct, but it is in the module output, isn't it? This is why I added this in the description.

Reply:
Let's track it separately. I need to get this on merged as functionality missing is a blocker